README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://gitlab.com/Northern.tech/Mender/mender-convert/badges/next/pipeline.svg)](https://gitlab.com/Northern.tech/Mender/mender-convert/pipelines)
 
-# mender-convert
+# Mender-convert
```

Comment:
Do not know if I agree that this this be upper cased. The tool name is `mender-convert ` and we have been fairly consistent with this.

Also drawing a parallel to `mender-artifact` where we have not tried to upper case that either.

So would drop this :)

Reply:
Alright, I figured this would be downvoted :p 
The idea was to uppercase the name of the tool, and then refer to the script itself in lower case.

Reply:
I agree with @mirzak here: Mender (the product, the company), but `mender-convert`, `mender-artifact`, `mender-cli` (the tools).